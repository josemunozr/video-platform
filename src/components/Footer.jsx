import React from 'react';
import '../assets/styles/components/Footer.scss';
import { Link } from 'react-router-dom';

const Footer = () => (
  <footer className='footer'>
    <Link to='/not-found'>Terminos de uso</Link>
    <Link to='/not-found'>Declaraci√≥n de privacidad</Link>
    <Link to='/not-found'>Centro de ayuda</Link>
  </footer>
);

export default Footer;
