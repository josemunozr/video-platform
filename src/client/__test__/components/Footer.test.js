import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
  const footer = mount(
    <MemoryRouter>
      <Footer />
    </MemoryRouter>
  );
  test('render Footer compoent', () => {
    expect(footer.length).toEqual(1);
  });

  test('Footer have 3 Link', () => {
    expect(footer.find('Link')).toHaveLength(3);
  });

  test('Footer snapshot', () => {
    const footer = create(
      <MemoryRouter>
        <Footer />
      </MemoryRouter>
    );
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
