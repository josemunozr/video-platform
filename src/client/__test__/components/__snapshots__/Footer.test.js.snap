// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Footer snapshot 1`] = `
<footer
  className="footer"
>
  <a
    href="/not-found"
    onClick={[Function]}
  >
    Terminos de uso
  </a>
  <a
    href="/not-found"
    onClick={[Function]}
  >
    Declaraci√≥n de privacidad
  </a>
  <a
    href="/not-found"
    onClick={[Function]}
  >
    Centro de ayuda
  </a>
</footer>
`;
